### ISSUE TITLE
Lattice graph not cleared with purge

### ISSUE THREAD

**Issue by dwelsch-memverge** [AgnostiqHQ/covalent#1434]
### Environment

- **Covalent version**: 0.202.0.post1
- **Python version**: 3.9.13
- **Operating system**: MacOS Moneteray 12.6 on Apple M1
- **Browser**: Firefox 107.0

### What is happening?

The UI seems to store a lattice graph even after all workflows have been purged and the server restarted.

### How can we reproduce the issue?

Create a lattice graph in a script using `[lattice].draw()`.
View the graph in the UI at localhost:48008

![Screen Shot 2022-12-01 at 9 16 38 AM](https://user-images.githubusercontent.com/58859892/205122793-60f494a9-3c4e-49f0-bf12-a1f329bb6856.png)

Stop the covalent server: covalent stop
Purge the covalent server: covalent purge -Hy
Start the covalent server: covalent start
Click the "Lattice draw preview" button

Result: The previously displayed transport graph is displayed, even though all workflows have been purged from the db.

### What should happen?

Presumably an exception should occur and the UI should tell you that no graph is available.

**Comment by santoshkumarradha** [comment]
@Prasy12 can we add this as backlog ?

**Comment by Prasy12** [comment]
@santoshkumarradha These are my findings related to this bug. 

1. Lattice draw or lattice preview is for the user to see how the transport graph would look on the UI at a draft stage with appropriate metadata. These are dispatches which haven't run. 
2. For the above reason, we don't store this draft data anywhere on the database and only on browser cache which is why the user was still able to see the graph when running covalent purge -Hy. (Purge removes files and DB) 

With the above information, can you let me know if that data also needs to be removed when a user runs purge? Since it is an draft data, does it add value maintaining that on the UI even after purge? 

cc : @mshkanth

**Comment by santoshkumarradha** [comment]
> @santoshkumarradha These are my findings related to this bug. 
> 
> 
> 
> 1. Lattice draw or lattice preview is for the user to see how the transport graph would look on the UI at a draft stage with appropriate metadata. These are dispatches which haven't run. 
> 
> 2. For the above reason, we don't store this draft data anywhere on the database and only on browser cache which is why the user was still able to see the graph when running covalent purge -Hy. (Purge removes files and DB) 
> 
> 
> 
> With the above information, can you let me know if that data also needs to be removed when a user runs purge? Since it is an draft data, does it add value maintaining that on the UI even after purge? 
> 
> 
> 
> cc : @mshkanth
> 
> 
> 
> 

Exactly. I don't see value in having it in UI. Can you have a backlog to take the logic of purging clearing that part of browser cache as well?
